gobblin-salesforce/src/main/java/org/apache/gobblin/salesforce/SalesforceConfigurationKeys.java

```diff
@@ -17,9 +17,24 @@
 
 package org.apache.gobblin.salesforce;
 
-public class SalesforceConfigurationKeys {
+public final class SalesforceConfigurationKeys {
+  private SalesforceConfigurationKeys() {
+  }
   public static final String SOURCE_QUERYBASED_SALESFORCE_IS_SOFT_DELETES_PULL_DISABLED =
       "source.querybased.salesforce.is.soft.deletes.pull.disabled";
-  public static final int DEFAULT_SALESFORCE_MAX_CHARS_IN_FILE = 200000000;
-  public static final int DEFAULT_SALESFORCE_MAX_ROWS_IN_FILE = 1000000;
+  public static final int DEFAULT_FETCH_RETRY_LIMIT = 5;
+  public static final String BULK_API_USE_QUERY_ALL = "salesforce.bulkApiUseQueryAll";
+
+  // pk-chunking
+  public static final String PK_CHUNKING_TEST_BATCH_ID_LIST = "salesforce.pkchunking.test.batch.id.list";
+  public static final String PK_CHUNKING_TEST_JOB_ID = "salesforce.pkchunking.test.job.id";
+  public static final String SALESFORCE_PARTITION_TYPE = "salesforce.partitionType";
+  public static final String PARTITION_PK_CHUNKING_SIZE = "salesforce.partition.PkChunkingSize";
```

Comment:
Should be "salesforce.partition.pkChunkingSize"

Reply:
thanks! fixed. good catch!